src/bandersnatch_filter_plugins/filename_name.py

```diff
@@ -0,0 +1,94 @@
+import logging
+from typing import List
+
+from bandersnatch.filter import FilterFilenamePlugin
+
+logger = logging.getLogger("bandersnatch")
+
+_plugin_initialized: bool = False
+_patterns: List[str] = []
+_packagetypes: List[str] = []
+
+
+def _init_once(self):
+    global _plugin_initialized
+
+    if _plugin_initialized:
+        return
+
+    try:
+        for platform in self.configuration["blacklist"]["platforms"].split("\n"):
+
+            if platform.lower() in ("windows", "win"):
```

Comment:
Can you set a var and call it lplatform to ensure always lowered rahter than calling it ever if/else

```
lplatform = platform.lower
if lplatform in ("bla", "ba"):
...
```

Reply:
ok.